test/test_certdirectory.py

```diff
@@ -261,6 +261,14 @@ def test_list_for_product_match(self):
         res = self.d.list_for_product('123456789')
         self.assertTrue(isinstance(res, list))
 
+    def test_list_for_product_name(self):
+        res = self.d.list_for_product_name('123')
+        self.assertTrue(isinstance(res, list))
```

Comment:
Better use assertIsInstance(), as it's available.

Reply:
Fixing.